### PR TITLE
Dashboard for all pages/routes demonstration

### PR DESCRIPTION
**Pull request by oce4ka**
Demo: https://deploy-preview-89--warm-frangollo-93bfd5.netlify.app/dashboard

Paths have been changed to shorter ones.
Dashboard page with all the paths and short annotations has been added.

### CODE REVIEW

**Review by Enigma2017 — APPROVED** [review]
Everything works, and these changes are available for merge

File: src/components/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    { path: '/delete', label: 'Delete User Dialog Example' },
+    { path: '/invite', label: 'Invitation' },
+    { path: '/error-400', label: 'Error 400' },
+    { path: '/error-500', label: 'Error 500' },
+    { path: '*', label: 'Error Page (Catch-All)' },
+    { path: '/new-match', label: 'New Match' },
+  ]
+
+  const routesTemporary = [
+    { path: '/account', label: 'User Account' },
```
Comment (Svitlana-G) [inline src/components/dashboard/Dashboard.tsx:27]: I propose to add some information for page account: in order for the page to work, you need to log in, have a profile saved in DB and launch the backend.

File: src/components/dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { Box, Button, Typography } from '@mui/material'
+
+const Dashboard = () => {
+  const routesNotAuthed = [{ path: '/', label: 'Register/Login page' }]
+  const routesAuthed = [
+    { path: '/callback', label: 'Auth Callback' },
```
Comment (Svitlana-G) [inline src/components/dashboard/Dashboard.tsx:7]: In my opinion, we don't need this route. It is used during registration. If we just click on this link, we will only see the logo

File: src/components/dashboard/Dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+          <Button
+            variant="text"
+            href={route.path}
+            sx={{ textDecoration: 'none', color: '#007bff' }}
+          >
+            {route.label}
+          </Button>
+        </Box>
+      ))}
+      <Typography variant="h2">Technical/temporary routes</Typography>
```
Comment (Svitlana-G) [inline src/components/dashboard/Dashboard.tsx:60]: We have the same title here and below. Maybe it makes sense to write something like this here: Technical/temporary routes for some components

**Review by Svitlana-G — APPROVED** [review]

**Pull request merged**